app/__init__.py

```diff
@@ -30,21 +33,19 @@ def create_app(config_name):
         configs[config_name],
         bootstrap=bootstrap,
         data_api_client=data_api_client,
-        feature_flags=feature_flags
+        feature_flags=feature_flags,
+        login_manager=login_manager
     )
     # Should be incorporated into digitalmarketplace-utils as well
     csrf.init_app(application)
 
+    application.permanent_session_lifetime = timedelta(hours=1)
```

Comment:
An hour is probably fine here too, but might we want admin app to be shorter than for suppliers?  I guess if it is locked down to AH and only Nicole as a user this will be fine though.